kubernetes/zenko/charts/burry/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for elasticsearch-curator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  suspend: true
+  # Every 5 minutes
+  schedule: "5 * * * *"
+  annotations: {}
+
+pod:
+  annotations: {}
+
+
+# serviceAccount:
+#   # Specifies whether a ServiceAccount should be created
+#   create: true
+#   # The name of the ServiceAccount to use.
+#   # If not set and create is true, a name is generated using the fullname template
+#   name:
+
+
+# psp:
+#   # Specifies whether a podsecuritypolicy should be created
+#   create: false
+
+image:
+  repository: zenko/burry
+  tag: 0.0.2
+  pullPolicy: IfNotPresent
+
+# hooks:
```

Comment:
What do these do?

Reply:
I left them for the future. I will add them back once I do that part.
Removed all unnecessary options